plugins/network-elements/nuage-vsp/src/com/cloud/network/manager/NuageVspManager.java

```diff
@@ -119,8 +121,8 @@
 
     /**
      * Associates a Nuage Vsp domain template with a
-     * @param cmd
-     * @return
+     * @param cmd Associate cmd which contains all the data
+     * @return true on success otherwise exception is thrown
```

Comment:
if only true can be returned why return? it is then pass or exception, right?

Reply:
@DaanHoogland that's correct. I will change it.